### DISCUSSION TITLE
Error : Please include tax rate in sales taxes and charges .

### DISCUSSION THREAD

**Discussion by BOTSolutions8686** [lavaloon-eg/ksa_compliance#133] [Q&A] [answered]
I am receiving this error in compliance . Although the tax rates are properly linked with tax catagory 
i.e. Standard Rate tax category linked with Sales taxes and charges template -> KSA VAT15% 

And guidance here will be appreciated.

**Comment by mhaggag** [comment]
Hello,

This error indicates there are no taxes on the invoice (`taxes` field, which is the `Sales Taxes and Charges` table). Do you have a default `Sales Taxes and Charges Template` configured for the company in question?

**Comment by BOTSolutions8686** [comment, marked as answer]
Yes . The tax rates were added properly but what the issue was that none of the sales taxes and charges were set as default. 
Everything worked fine when we set one of them to default
